tests/operators/test_latest_only_operator.py

```diff
@@ -25,7 +25,7 @@
 from airflow import settings
 from airflow.models import DagRun, TaskInstance
 from airflow.operators.empty import EmptyOperator
-from airflow.operators.latest_only import LatestOnlyOperator
+from airflow.providers.standard.operators.latest_only import LatestOnlyOperator
```

Comment:
This file should move under standard provider

Reply:
@romsharon98 Thanks for pointing that out. I have moved the file now.

Reply:
Some tests are failing with the error below after moving `test_latest_only_operator.py` file to Standard providers. 

`providers/tests/standard/operators/test_latest_only_operator.py::TestLatestOnlyOperator::test_skipping_non_latest - sqlalchemy.exc.ArgumentError: ORDER BY expression expected, got <property object at 0x7fef5a963770>.`

I tried running the test locally but can't seem to reproduce the error as these tests are passing. Would love to get your opinions on why this might be happening. Thanks.

Reply:
Those are backwards compatibility testing (See the https://github.com/apache/airflow/blob/main/contributing-docs/testing/unit_tests.rst#compatibility-provider-unit-tests-against-older-airflow-releases - it contains information how to reproduce them locally and some typical ways of handling back-compatibility

Reply:
@potiuk  Thanks for the pointers. After reading the documentation and testing locally I see that the test failure is due to the changes introduced by this PR #43902 where `execution_date` was changed to `logical_date`. I have added a condition to skip these failing tests for versions less than 3.0. 

Please let me know if this is the right thing to do.